src/images/index.js

```diff
@@ -8,7 +8,12 @@ import go from "./go.png";
 import electron from "./electron.png";
 import next from "./next.png";
 import gucci from "./gucci.jpg";
+import gitTrends from "./gitTrends.svg";
 import graphql from "./graphql.png";
```

Comment:
Seems like the images exports got broken, this should fix that
```suggestion
import gucci from "./gucci.jpg";
import gitTrends from "./gitTrends.svg";
```